sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 
 #### Bugs Fixed
 * Fixed an issue where `emptyPageDiagnosticsEnabled` in `CosmosQueryRequestOptions` was being overridden. This caused empty page diagnostics to be logged (with INFO level) even when the flag was set to false - See [PR 37199](https://github.com/Azure/azure-sdk-for-java/pull/37199)
+* Added 410/1002 handling for `ChangeFeedProcessor#getCurrentState` in **Latest Version**, **All Version and Deletes** changes modes. - See [PR 37107](https://github.com/Azure/azure-sdk-for-java/pull/37107)
```

Comment:
since this PR only targets for EPK-Range based leases, would be good to call it out

Reply:
ah got it, it is mentioned in the note